include/velocypack/Slice.h

```diff
@@ -526,7 +582,7 @@ class Slice {
   // returns the Slice managed by an External or the Slice itself if it's not
   // an External, recursive version
   Slice resolveExternals() const {
-    uint8_t const* current = _start;
+    uint8_t const* current = valueStart();
```

Comment:
I think this should still operate on `start()` and not on `valueStart()`.
If a client application needs to peek into the value behind the tag, I think it should explicitly resolve the tag.